.ci/docker/node-playwright/Dockerfile

```diff
@@ -0,0 +1,53 @@
+# Little bit customized version of microsoft/playwright:bionic image
+
+FROM ubuntu:bionic
+
+# 1. Install node12
+RUN apt-get update && apt-get install -y curl && \
+    curl -sL https://deb.nodesource.com/setup_12.x | bash - && \
+    apt-get install -y nodejs
+
+# 2. Install git (used to tag commit in benchmark runner)
+RUN apt-get install -y git --no-install-recommends
+
+# 3. Install WebKit dependencies
+RUN apt-get install -y libwoff1 \
+                       libopus0 \
+                       libwebp6 \
+                       libwebpdemux2 \
+                       libenchant1c2a \
+                       libgudev-1.0-0 \
+                       libsecret-1-0 \
+                       libhyphen0 \
+                       libgdk-pixbuf2.0-0 \
+                       libegl1 \
+                       libnotify4 \
+                       libxslt1.1 \
+                       libevent-2.1-6 \
+                       libgles2 \
+                       libvpx5
+
+# 4. Install Chromium dependencies
+RUN apt-get install -y libnss3 \
+                       libxss1 \
+                       libasound2
+
+# 5. Install Chrome unstable to run karma benchmark tests inside puppeteer
+RUN apt-get update \
+    && apt-get install -y wget --no-install-recommends \
+    && wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
+    && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
+    && apt-get update \
+    && apt-get install -y \
+        google-chrome-unstable \
+        fonts-ipafont-gothic \
+        fonts-wqy-zenhei \
+        fonts-thai-tlwg \
+        fonts-kacst \
+        --no-install-recommends \
+        && rm -rf /var/lib/apt/lists/*
+
+# 6. Install Firefox dependencies
+RUN apt-get update \
+    && apt-get install -y libdbus-glib-1-2 \
+                          xvfb
```

Comment:
```suggestion
RUN apt-get install -y libdbus-glib-1-2 \
                          xvfb

```

Reply:
I was getting these errors if i dont update 

```
14:09:44  Service 'node-benchmark' failed to build: The command '/bin/sh -c apt-get install -y libdbus-glib-1-2                           xvfb' returned a non-zero code: 100
```

Reply:
Sure thing that was caused by https://github.com/elastic/apm-agent-rum-js/pull/683/files#diff-421113117290176fb1a618548b8c9943R48